bindings/ergo-lib-c-core/src/rest/api/callback.rs

```diff
@@ -0,0 +1,62 @@
+use std::ffi::c_void;
+use std::ptr;
+use std::ptr::NonNull;
+
+use ergo_lib::ergo_rest::NodeResponse;
+
+use crate::Error;
+
+/// Callback info from the Swift side
+#[repr(C)]
+pub struct CompletionCallback {
+    /// Wrapped Swift closure pointer (see WrapClosure.swift)
+    swift_closure: NonNull<c_void>,
```

Comment:
If these are `swift` only should we put it behind a feature flag so it's not included in C?

Reply:
Good question! It got me thinking that there is actually nothing inherently specific to Swift. So I renamed both callback structs and their fields and it made names and comments so much clearer! Thanks!
Here is how they look now: https://github.com/ergoplatform/sigma-rust/blob/500ad8b867ee3c882fec7ccccee5993260b05c49/bindings/ergo-lib-c-core/src/rest/api/callback.rs#L9-L20